dbsnp-import/src/main/java/uk/ac/ebi/eva/server/models/ProgressReport.java

```diff
@@ -56,12 +56,18 @@
     @Convert(converter = StatusConverter.class)
     private Status variantsImported;
 
+    @Enumerated(EnumType.STRING)
+    @Convert(converter = StatusConverter.class)
+    private Status variantsWithoutEvidenceImported;
```

Comment:
I don't like so much the name variantsWithoutEvidenceImported. I cannot think of a btter alternative either... Some suggestions:
- legacyVariantsImported... (but the ones with evidence are also "legacy".)
- unconfirmedVariantsImported 
- unprovedVariatsImported 

This is an internal name anyway, so I'm OK if nobody else can think of a better name.

Reply:
We could name the variables the other way around: `variantsWithEvidence` for the ones we already have imported, and just `variants` for them in general. I think it is more clear because `variantsImported` seems to refer to all of them, possibly including those without evidence.

Reply:
I like it! That is "darle la vuelta a la tortilla"!